airbyte-integrations/connectors/source-jira/source_jira/entities.py

```diff
@@ -0,0 +1,133 @@
+"""
+MIT License
+
+Copyright (c) 2020 Airbyte
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+# generated for keeping entities base params
+ENTITIES_MAP = {
+    "application_roles": {"url": "/applicationrole", "func": lambda v: v, "params": {}},
+    "avatars": {"func": lambda v: v["system"], "params": {}},
+    "dashboards": {"url": "/dashboard", "func": lambda v: v["dashboards"], "params": {}},
+    "filters": {"url": "/filter/search", "func": lambda v: v["values"], "params": {}},
+    "filter_sharing": {"func": lambda v: v, "params": {}},
```

Comment:
it's confusing that some entities don't have the `url` property set. It's also not good from a DRY point of view because this object is supposed to be the source of truth for the URL. Could we instead always include `url` in this object (or at least as much as we can), and in the consuming context use `url.format(params)`?